.github/workflows/build-and-test.yml

```
@@ -13,31 +13,29 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Link shellcheck into expected location
+        run: sudo ln -s $(which shellcheck) /usr/local/bin/shellcheck
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.10'
+      - name: Install Black
+        run: pip install black
```

```suggestion
      - name: Set up Python
        uses: actions/setup-python@v1
        with:
          python-version: '3.10'
      - name: Install black and link shellcheck into expected location
        run: |
          pip install black
          sudo ln -s $(which shellcheck) /usr/local/bin/shellcheck
```

These steps can potentially be simplified as follows.